docs/chart_template_guide/functions_and_pipelines.md

```diff
@@ -150,7 +150,7 @@ Template functions and pipelines are a powerful way to transform information and
 
 ## Operators are functions
 
-Operators are implemented as functions that return a boolean value. To use `eq`, `ne`, `lt`, `gt`, `and`, `or`, `not` etcetera place the operator at the front of the statement followed by its parameters just as you would a function. To chain multiple operations together, separate individual functions by surrounding them with paranthesis.
+Operators are implemented as functions that return a boolean value. To use `eq`, `ne`, `lt`, `gt`, `and`, `or`, `not` etcetera place the operator at the front of the statement followed by its parameters just as you would a function. To chain multiple operations together, separate individual functions by surrounding them with parenthesis.
```

Comment:
parenthesis --> parentheses

Reply:
Thank you very much, this has been updated as your recommendation.

Reply:
@tuanvcw I do not see the changes

Reply:
@hickeyma, sorry I forgot to push the new change.
The patch has just been updated, thanks a lot :smile:

Reply:
Not a bother